lib/internal/readline/interface.js

```diff
@@ -1052,9 +1052,8 @@ class Interface extends InterfaceConstructor {
   }
 
   // Handle a write from the tty
-  [kTtyWrite](s, key) {
+  [kTtyWrite](s, key = kEmptyObject) {
```

Comment:
The pull request still has irrelevant changes.

Reply:
Oops, I thought I caught them all, I'll do a double take